docs/optimization.md

```diff
@@ -283,6 +283,22 @@ cutting latency in half. The benefit provided by TensorRT will vary
 based on the model, but in general it can provide significant
 performance improvement.
 
+### ONNX with OpenVINO Optimization
+
+ONNX models running on the CPU can also be accelerated by using
+[OpenVINO](https://docs.openvinotoolkit.org/latest/index.html).  To
+enable OpenVINO optimization for an ONNX model: stop Triton, add the
```

Comment:
We should re-word "stop Triton" to "before launching Triton" for the sake of clarity.